portality/scripts/all_returning_500_via_api.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+from portality import models
+from portality.core import app
+import esprit
+import re
+import csv
+
+INVALID_URLS = {
+        "query": {
+            "match_all": {}
+        }
+}
+
+if __name__ == "__main__":
+
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--out", help="output file path")
+    args = parser.parse_args()
+
+    if not args.out:
+        print("Please specify an output file path with the -o option")
```

Comment:
Argparse has its own `required` which will do exactly this https://docs.python.org/3/library/argparse.html#required